Donation-platform/src/main/webapp/personalizationScript.js

```diff
@@ -1,103 +1,103 @@
-function updatePersonalizedList(charities) {
-  const personalizedListElement = document.getElementById('personalized-list');
+// /* (2) Displays the list of personalized charities them on personalized.html */
+// function updatePersonalizedList(charities) {
+//   const personalizedListElement = document.getElementById('personalized-list');
 
-  // Remove old personalized results
-  personalizedListElement.innerHTML = "";
+//   // Remove old personalized results
+//   personalizedListElement.innerHTML = "";
   
-  // Displays charity listings in the format: "[Charity Name] ([tag 1], [tag2], ...)"
-  charities.forEach(charity => {
-    var charityString = "<li><strong>" + charity.name + "</strong>  (";
-    for(i = 0; i < (charity.tags).length; i++) {
-      if(i == (charity.tags).length - 1) {
-        charityString += (charity.tags)[i] + ")</li>";
-      } else {
-        charityString += (charity.tags)[i] + ", ";
-      }
-    }
-    personalizedListElement.innerHTML += charityString;
-  });
-}
+//   // Displays charity listings in the format: "[Charity Name] ([tag 1], [tag2], ...)"
+//   charities.forEach(charity => {
+//     var charityString = "<li><strong>" + charity.name + "</strong>  (";
+//     for(i = 0; i < (charity.tags).length; i++) {
+//       if(i == (charity.tags).length - 1) {
+//         charityString += (charity.tags)[i] + ")</li>";
+//       } else {
+//         charityString += (charity.tags)[i] + ", ";
+//       }
+//     }
+//     personalizedListElement.innerHTML += charityString;
+//   });
+// }
 
-/**
- * Updates the display with bootstrap cards.
- */
-function updatePersonalizedCardsOnPage(charities) {
-    const cards = document.getElementById('cards');
+// /**
+//  * Updates the display with bootstrap cards.
+//  */
+// function updatePersonalizedCardsOnPage(charities) {
+//     const cards = document.getElementById('cards');
 
-    cards.innerHTML = '';
+//     cards.innerHTML = '';
 
-    charities.forEach(charity => {
-        cards.innerHTML += '<div class="card border-dark mb-3">' + '<img class="card-img-top" src=' + charity.imgSrc + ' alt="Card image">' + '<div class="card text-center">' + '<h4 class="card-title">' + charity.name + '</h4>' + 
-                            '<p class="card-text">' + displayTags(charity.tags) + '</p>' + '<a href=' + charity.link + ' target=_blank class="btn btn-primary" role="button">Donate</a>';
-        cards.innerHTML += '</div>' + '</div>';
-    });
-}
+//     charities.forEach(charity => {
+//         cards.innerHTML += '<div class="card border-dark mb-3">' + '<img class="card-img-top" src=' + charity.imgSrc + ' alt="Card image">' + '<div class="card text-center">' + '<h4 class="card-title">' + charity.name + '</h4>' + 
+//                             '<p class="card-text">' + displayTags(charity.tags) + '</p>' + '<a href=' + charity.link + ' target=_blank class="btn btn-primary" role="button">Donate</a>';
+//         cards.innerHTML += '</div>' + '</div>';
+//     });
+// }
```

Comment:
why is everything commented out

Reply:
fixed that. the comments were left over from when I was trying to get my tests to run locally.